assets/scss/components/_footer.scss

```diff
@@ -2,18 +2,181 @@
    #FOOTER
    ========================================================================== */
 
-footer {
-    background-color: $color-white;
+$at-sign-icon: '../../img/icons/at-sign.svg';
+
+.footer {
+    background-color: $color-secondary-50;
     border-top: 1px solid $color-white-grey;
-    padding: 3rem 0;
+    padding: 3rem;
     position: relative;
     transform: translateZ(0);
 
-    @include bp(medium) {
-        section {
+    section {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 1.5rem;
+
+        @include bp(small) {
             display: grid;
             gap: remcalc(10.5);
-            grid-template-columns: 1fr 1fr 1fr auto;
+            grid-template-columns: repeat(3, 1fr);
+        }
+    }
+
+    @include bp(medium) {
+        grid-template-columns: repeat(4, 1fr);
+        padding: 5rem;
+    }
+}
+
+.footer-column {
+    width: calc(50% - 0.75rem);
+
+    &:nth-child(3) {
+        width: 100%;
+    }
+}
+
+.footer-container {
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+
+    @include bp(medium) {
+        gap: 3rem;
+    }
+}
+
+.footer-heading {
+    color: $color-black;
+    font-family: $font-regular;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.5rem;
+
+    margin-bottom: 0.75rem;
+}
+
+.footer-links {
+    margin: 0;
+}
+
+.footer-link-item {
+    list-style-type: none;
+    margin-bottom: 0.5rem;
+}
+
+.footer-link {
+    color: $color-primary-300;
+    font-family: $font-regular;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.5;
+    text-decoration-line: underline;
+}
+
+.newsletter-subscription {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    grid-column: 1 / span 2;
+
+    width: 100%;
+    max-width: 30.375rem;
+}
+
+.newsletter-heading {
+    color: $color-black;
+    font-family: $font-regular;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 1.5;
+
+    margin: 0;
+}
+
+.newsletter-text {
+    color: $color-neutrals-400;
+    font-family: $font-regular;
+    font-size: 1rem;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 1.5;
+
+    margin: 0;
+    max-width: 21.375rem;
+}
+
+.newsletter-fields {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 0.5rem;
+
+    .c-input {
```

Comment:
Avoid nesting if possible. Instead, create another modifier class for `c-input` so all `c-input` styles will be in one place. 

Alternatively, if you must to nest selectors, keep component's styles in one place:
`.c-newsletter-field .c-input { ... }` in `components/input` file